int8 meta10
